http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorRoutes.scala

```diff
@@ -19,51 +18,41 @@ class CollectorRoutes[F[_]: Sync](collectorService: Service[F]) extends Http4sDs
 
   private val cookieRoutes = HttpRoutes.of[F] {
     case req @ (POST | GET | HEAD) -> Root / vendor / version =>
```

Comment:
I like how with these new changes the routes class is very clean and visually easy to read.

What do you think about splitting out the `/vendor/version` endpoint into separate POST and GET -- would this look even more clean?

```scala
  case req @ POST -> Root / vendor / version =>
    ???

  case req @ (GET | HEAD) -> Root / vendor / version =>
    ???

  case case req @ (GET | HEAD) -> Root / ("ice.png" | "i") =>
    ???
```